python/sglang/bench_latency.py

```diff
@@ -340,13 +340,16 @@ def latency_test_run_once(
             rank_print(
                 f"Decode.  latency: {latency:6.5f} s, throughput: {throughput:9.2f} token/s"
             )
-    med_decode_latency = np.median(decode_latencies)
-    med_decode_throughput = batch_size / med_decode_latency
-    rank_print(
-        f"Decode.  median latency: {med_decode_latency:6.5f} s, median throughput: {med_decode_throughput:9.2f} token/s"
-    )
-    measurement_results["median_decode_latency"] = med_decode_latency
-    measurement_results["median_decode_throughput"] = med_decode_throughput
+
+    # record decode timing from 2nd output
+    if output_len > 1:
```

Comment:
Nit:

From the PoV of semantics, the check `if output_len > 1` should go ahead of the line `decode_latencies = []` because the decoding phase won't happen unless the specified arg (one of the values of `BenchArgs.output_len`) is greater than 1.